### PR TITLE
Ruby group capstone - Catalog of my things

### PR DESCRIPTION
**Pull request by rolandfbk**
### In this milestone, we performed the following tasks:

- Insert and query data in SQL.
- Use primary key & foreign key mechanisms for joining tables.
- Understand the different types of relationships between tables.
- Query multiple tables.
- Prepare complex queries that answer analytical questions.
- Run a program using the command line.
- Use Ruby syntax for basic programming operations.
- Apply Ruby best practices and language style guides in code.
- Describe the SOLID principles of OOP.
- Implement classes and objects in Ruby.
- Understand the four main principles of OOP.
- Implement encapsulation and inheritance with Ruby.
- Create a UML class diagram.
- Explain the difference between associations, aggregations, and composition in OOP.
- Set up associations between classes and objects.
- Store data in files.
- Build interactive console apps.
- Recognize the value of making equal contributions to group projects to produce the best outcome.

**[Link to the video presentation](https://drive.google.com/file/d/1-ZtiRJjZaeBX-GUzWybjcgw_GmhwQM6o/view?usp=sharing
)**

### CODE REVIEW

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @rolandfbk, @jasonfelice, @Mikelobi 🙋‍♀️,  

Great work so far! 💯 
You have done a great job implementing most of the project requirements for the Catalog of My Things app. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  All important classes and methods are present
✔️  Relationships and associations based on the [UML diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/groupl-capstone/images/catalog_of_my_things.png) implemented
✔️  Data is preserved using JSON
✔️  Tests are written and they all pass
✔️  Good PR title and description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
[the remark this answers is not in the document]
Comment (rotimiazeez) [reply README.md:65]: - As requested by the previous reviewer, kindly add a valid link to your license as seen [here](https://github.com/microverseinc/readme-template) 👍🏽.

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+- GitHub: [@Mikelobi](https://github.com/Mikelobi)
+- Twitter: [@Mikelobi](https://twitter.com/omulum)
+- LinkedIn: [@Mikelobi](https://www.linkedin.com/in/ugochukwu-omulu/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
[the remark this answers is not in the document]
Comment (rotimiazeez) [reply README.md:52]: - Please make sure a valid `issue` link is present as requested. 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi TEAM 👏🏽, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply README.md:65]: ## Optional
- [ ] I suggest that you correct the link or remove it in the README

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply README.md:52]: ## Optional
- [ ] Please correct the link or remove it

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**